readme.md

```diff
@@ -51,38 +51,19 @@ Default: `false`
 
 Crop to the set height.
 
-##### delay
-
-Type: `number` *(seconds)*<br>
-Default: `0`
-
-Delay capturing the screenshot. Useful when the site does things after load that you want to capture.
-
 ##### timeout
 
 Type: `number` *(seconds)*<br>
 Default: `60`
 
-Number of seconds after which PhantomJS aborts the request.
+Number of seconds after which Google Chrome aborts the request.
 
 ##### selector
 
 Type: `string`
 
 Capture a specific DOM element.
 
-##### css
-
-Type: `string`
-
-Apply custom CSS to the webpage. Specify some CSS or the path to a CSS file.
-
-##### script
```

Comment:
We could use [`page.addScriptTag(url)`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageaddscripttagurl).

Reply:
Yup. In conjunction with [`injectFile(path)`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageinjectfilefilepath).